src/styles/global.css

```diff
@@ -2,8 +2,9 @@
 @import 'tw-animate-css';
 
 @custom-variant dark (&:is(.dark *));
-
 :root {
+  --default-font-family: 'Inter', sans-serif;
```

Comment:
For this change, we would need to load the fonts?

Reply:
no, this relies on system fonts for now